src/OrleansRuntime/Placement/IPlacementContext.cs

```diff
@@ -26,6 +26,8 @@ internal interface IPlacementContext
 
         SiloAddress LocalSilo { get; }
 
+        SiloStatus LocalSiloStatus { get; }
```

Comment:
Ok, good to know.

Doesn't matter here as far as I can tell, it's all internal.